src/Components/SampleModelsControl.jsx

```diff
@@ -134,13 +134,13 @@ const useStyles = makeStyles({
   root: {
     'width': '260px',
     '& .MuiOutlinedInput-input': {
-      color: 'green',
-    },
-    '& .MuiInputLabel-root': {
-      color: 'green',
+      color: 'blue',
     },
+    // '& .MuiInputLabel-root': {
```

Comment:
@pablo-mayrgundter temporarily commenting it out - to be picked up by the theme.